File: 6_delete_object.py
class Employee: 
    depatment = "IT"
    
    def show(self):
        print('Department is :', self.depatment)
emp = Employee()
emp.show()

# delete the Object 
del emp 
#accesing the object after deleting it will throw an error
emp.show()
# output: NameError: name 'emp' is not defined

File: 6_delete_object.php
<?php 
class Employee {
    // Define the $department property as a class variable
    public $department = "IT";

    // Define the show() method to print the department property
    public function show() {
        echo 'Department is : '.$this->department;
    }
}

$emp = new Employee();
$emp->show();

unset($emp);
$emp->show(); // Will throw an error, since $emp is no longer defined